### PR TITLE
#10

### PR DESCRIPTION
**Pull request by prymas007**

### CODE REVIEW

File: src/PHPUnitRandomizer/Randomizer.php [hunk 1]
```diff
@@ -4,48 +4,47 @@
...
-	/**
-	 * Order the TestSuite tests in a random order.
-	 *
-	 * @param  \PHPUnit_Framework_Test $suite The suite to randomize.
-	 * @param  integer                 $seed  Seed used for PHP to randomize the suite.
-	 * @return \PHPUnit_Framework_Test
-	 */
-	public function randomizeTestSuite(\PHPUnit_Framework_Test $suite, $seed)
-	{
+    /**
```
Comment (fiunchinho) [inline src/PHPUnitRandomizer/Randomizer.php]: Could you keep the same coding style as the other files?

File: src/PHPUnitRandomizer/Randomizer.php [hunk 2]
```diff
@@ -57,34 +56,133 @@ private function testSuiteContainsOtherSuites($suite)
...
-     * @param  integer 					$order Arbitrary value to "salt" the seed.
+     *
+     * @param  \PHPUnit_Framework_Test     $suite Test suite to randomize.
+     * @param  integer                     $seed  Seed to be used for the random funtion.
+     * @param  integer                     $order Arbitrary value to "salt" the seed.
+     * @param  bool                        $fix_depends [=false]
      * @return \PHPUnit_Framework_Test
      */
-    private function randomizeSuite($suite, $seed, $order = 0)
+    private function randomizeSuite($suite, $seed, $order = 0, $fix_depends = false)
```
Comment (fiunchinho) [inline src/PHPUnitRandomizer/Randomizer.php]: I'd make it true by default in all methods.

**Comment by fiunchinho** [comment]
Thank you for the contribution! Can you fix the details I pointed out?

**Comment by prymas007** [comment]
yeah, will do

**Comment by prymas007** [comment]
pushed

**Comment by fiunchinho** [comment]
Thank you very much!

**Pull request merged**